python/ray/serve/scripts.py

```diff
@@ -208,85 +217,24 @@ def shutdown(address: str, namespace: str):
 
 
 @cli.command(
-    help="""
-[Experimental]
-Create a deployment in running Serve instance. The required argument is the
-import path for the deployment: ``my_module.sub_module.file.MyClass``. The
-class may or may not be decorated with ``@serve.deployment``.
-""",
-    hidden=True,
-)
-@click.argument("deployment")
-@click.option(
-    "--address",
-    "-a",
-    default=os.environ.get("RAY_ADDRESS", "auto"),
-    required=False,
-    type=str,
-    help='Address of the running Ray cluster to connect to. Defaults to "auto".',
-)
-@click.option(
-    "--namespace",
-    "-n",
-    default="serve",
-    required=False,
-    type=str,
-    help='Ray namespace to connect to. Defaults to "serve".',
-)
-@click.option(
-    "--runtime-env-json",
-    default=r"{}",
-    required=False,
-    type=str,
-    help=("Runtime environment dictionary to pass into ray.init. Defaults to empty."),
-)
-@click.option(
-    "--options-json",
-    default=r"{}",
-    required=False,
-    type=str,
-    help="JSON string for the deployments options",
-)
-def create_deployment(
-    address: str,
```

Comment:
this wasn't exposed at all

Reply:
I wasn't sure what to do with this. It's the old `serve deploy` CLI command, but I changed its name to `create-deployment`. I don't think it's in the docs, and it was always experimental, so I think we can safely remove it.